nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -162,6 +162,8 @@ Use `configure.packages` instead.
 
 - A new module was added for the Saleae Logic device family, providing the options `hardware.saleae-logic.enable` and `hardware.saleae-logic.package`.
 
+- There is a new module to allow configuring settings for the `ntfy.sh` service.
```

Comment:
It is in the wrong section, it should be in "New Services" and you should put links to the option

Reply:
Moved